reporting-app/app/models/activity.rb

```diff
@@ -9,6 +9,8 @@ class Activity < ApplicationRecord
 
   strata_attribute :month, :date
   strata_attribute :name, :string
+  strata_attribute :category, :string
 
   validates :name, presence: true
+  validates :category, inclusion: { in: %w[employment education community_service] }
```

Comment:
What do you think about making the category array a constant? 


```suggestion
  ALLOWED_CATEGORIES = %w[community_service education employment]
  ...
  validates :category, inclusion: { in: ALLOWED_CATEGORIES }
```

Then we can use this array to create the radio checkboxes, so those values are always in sync